rllib/algorithms/alpha_zero/tests/test_alpha_zero.py

```diff
@@ -23,6 +24,10 @@ def test_alpha_zero_compilation(self):
         """Test whether AlphaZero can be built with all frameworks."""
         config = (
             az.AlphaZeroConfig()
+            .resources(
+                # Use GPUs iff `RLLIB_NUM_GPUS` env var set to > 0.
+                num_gpus=float(os.environ.get("RLLIB_NUM_GPUS", "0"))
```

Comment:
Can we do a similar thing for consistency? i.e. make num_gpus a class attribute and use that here?